README.md

```
@@ -83,12 +87,42 @@ jobs:
       - uses: reactive-firewall/shellcheck-scan@v1
 ```
 
+### Full Usage
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)